index.html

```diff
@@ -28,25 +28,24 @@ <h3><button>Sign up</button></h3>
 
     <main>
         <section class="box1"></section>
-        
-        <section class="box3"></section>
-        <p>this is some subtext under an illustration or image</p>
         <div class="box2"><img src="./image/IMG-20220320-WA0110.jpg" alt="an animation" width="125px;" height="117px;"></div>
```

Comment:
Always name your images such that their names are meaningful and related to the context in which they are used. Kindly change the name of your images.

Reply:
Done.